test/e2e/suite_test.go

```diff
@@ -169,6 +174,7 @@ func cleanupTestNamespace(namespace string, kubeClient kubernetes.Interface) err
 	return nil
 }
 
+//nolint:unused
```

Comment:
shall we remove the unused code?

Reply:
Too much code needed to be removed and I committed it in a new commit.